src/SketchPad.jsx

```diff
@@ -48,9 +48,13 @@ export default class SketchPad extends Component {
   constructor(props) {
     super(props);
     this.initTool = this.initTool.bind(this);
+    this.useTouch = 'ontouchstart' in window;
```

Comment:
can you move this into `getCursorPosition`, so instead here it would be there like `if(!!window.ontouchstart) { ... }`